src/PolyhedralGeometry/iterators.jl

```diff
@@ -85,6 +85,30 @@ Return a `RayVector` resembling a ray from the origin through the point whose co
 """
 ray_vector
 
+function Base.:(==)(x::RayVector, y::RayVector)
+  ax = collect(x)
+  ay = collect(y)
+  ix = findfirst(!is_zero, ax)
+  iy = findfirst(!is_zero, ay)
+  ix == iy || return false
+  isnothing(ix) && return true
+  r = ay[iy]//ax[ix]
+  r < zero(r) && return false
+  return (r .* ax == ay)
```

Comment:
```suggestion
  ix = findfirst(!is_zero, x)
  iy = findfirst(!is_zero, y)
  ix == iy || return false
  isnothing(ix) && return true
  r = y[iy]//x[ix]
  is_negative(r) && return false
  return r * x.p == y.p
```
This should be a bit faster and avoid some allocations.